posturebest/Components/ConfigureAlertsTab.swift

```diff
@@ -48,18 +52,19 @@ struct ConfigureAlertsTab: View {
                 
             }.padding(.top, 0)
             HStack {
-                Text("Posture Reminder Frequency (mins)")
+                Text("Posture Reminder \nFrequency (mins)")
                     .font(.headline)
                     .foregroundStyle(Color(hex: "#374663"))
                     .padding(.top, 20)
-                // add i hover
+               
+                InfoButtonView(message: postureInfo, buttonSize: 15, title: "Posture Reminders").offset(x: -27, y: 20)
                 
                 Picker("Select Frequency", selection: $postureFreqState) {
```

Comment:
nit maybe or future thing to discuss, probably out of scope. Idk how I feel about the different time format for the two reminders (0:15 vs 15). Ik the standing reminder should be less frequent, but the different formatting is still bothering me lol. could we do both the same? Maybe standing could still be in minute format even if it goes over an hour (75 vs 1:15)

Reply:
I can add a small ticket to change the format since it doesn't rlly have to do with this ticket! we can discuss whether to actually have it go to 1 or just add a 0: in front?